contracts/src/v0.8/dev/functions/FunctionsBillingRegistry.sol

```diff
@@ -850,4 +862,11 @@ contract FunctionsBillingRegistry is
   function _canSetAuthorizedSenders() internal view override onlyOwner returns (bool) {
     return true;
   }
+
+  /**
+   * @dev This empty reserved space is put in place to allow future versions to add new
+   * variables without shifting down storage in the inheritance chain.
+   * See https://docs.openzeppelin.com/contracts/4.x/upgradeable#storage_gaps
+   */
```

Comment:
After reading through the docs, it looks like upgrades should be pretty easy using the provided OpenZepplin HardHat plugin, but I want to make sure we do a test upgrade prior to merging this change.

Reply:
Can we schedule a sync w/ the team to do a run-through of the upgrade process?  Ideally, we should all know how to implement a contract hotfix if necessary.